integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorWlsUpgrade.java

```diff
@@ -235,7 +225,6 @@ void testOperatorWlsAuxDomainUpgradeFrom340ToCurrent() {
    * Note testOperatorWlsAuxDomainUpgradeFrom340ToCurrent() uses a 
```

Comment:
Also, remove this comment since `testOperatorWlsAuxDomainUpgradeFrom340ToCurrent` is deleted.